### PR TITLE
UI: Update flight icons

### PR DESCRIPTION
**Pull request by hashishaw**
This PR updates our icon usage to eliminate the need for structure icons 🎉 

We still have 2 custom icons (`duo` and `pingid`) that prevent us from eliminating svg-jar completely. 

**Auth mount before / after**
auth-mount-comparison

**Engine mount before / after**
secrets-mount-comparison

**Dashboard secrets engine card also includes color icons when available**
dashboard-list-with-color

**KMIP role operations before / after**
operations-comparison

### CODE REVIEW

File: changelog/24823.txt
Comment (hashishaw) [inline changelog/24823.txt:1]: Not totally sure this PR needs a changelog, open to suggestions

Comment (Monkeychip) [reply changelog/24823.txt:1]: I think a changelog is a good idea. It's a visual enough change that some customers may notice and look for the cause.

File: ui/app/templates/vault/cluster/access/methods.hbs [hunk 1]
```diff
@@ -56,14 +56,7 @@
...
           <Hds::TooltipButton @text={{method.methodType}} aria-label="Type of auth mount">
-            <Icon
-              @name={{if
-                (or (find-by "type" method.methodType (mountable-auth-methods)) (eq method.methodType "token"))
-                method.methodType
-                "auth"
-              }}
-              class="has-text-grey-light"
-            />
+            <Icon @name={{method.icon}} class="has-text-grey-light" />
```
Comment (Monkeychip) [inline ui/app/templates/vault/cluster/access/methods.hbs:59]: Nice clean up here.

**Review by Monkeychip — APPROVED** [review]
Really great work!! Thank you.

**Pull request merged**